eldritchmush/commands/combat_commands/stagger.py

```diff
@@ -17,8 +17,10 @@ class CmdStagger(Command):
     key = "stagger"
     help_category = "mush"
 
+    def __init__(self):
```

Comment:
I think I see why you're running the __init__'s..though in the case of a target variable in the function, we handle for it universally.